portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -15,6 +15,52 @@
 /** Spaces out comment text from author name */
 const commentHyphen = ' -';
 
+/**
+ * Supported class states for showing and hiding containers.
+ * @enum {string}
+ */
+const states = {
+  SHOW: 'block',
+  HIDE: 'none',
+}
+
+getUserLoginData();
+
+/**
+ * Fetches user login data from servlet and adjusts comments section of portfolio
+ * to hide comments if user is logged out.
+ */
+async function getUserLoginData() {
+  const response = await fetch('/user-login');
+  const userData = await response.json();
+
+  var loginButtonContainer = document.getElementById('login-button-container');
+  var commentForm = document.getElementById('comment-form');
+  var loginButtonForm = document.getElementById('login-button-form');
+  var logoutButtonForm = document.getElementById('logout-button-form');
+  var lineBreak = document.getElementById('line');
+  var deleteButton = document.getElementById('delete-button');
+
+  /** Hide/show containers depending on user login state */
```

Comment:
So usually you do you create javadoc style comments for the inside of the method. You would do // or /* */

Reply:
done